src/components/SingleCountry.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const SingleCountry = ({ name, area }) => (
+  <Link
+    to={`country/${name}`}
+    className="shadow border-r border-b border-slate-300 text-white bg-primary h-[170px] box-border basis-1/2 flex flex-col justify-end items-end p-4"
+  >
+    <h3 className="text-xl text-right font-extrabold uppercase">{name}</h3>
+    <p className="text-base font-medium">
+      {area}
+      {' '}
+      km
+      <sup>2</sup>
+    </p>
+  </Link>
+);
```

Comment:
Adding the enter icon to each cell will make your work look better since you are not desplayin the maps 

![image (2)](https://github.com/martinkarugaba/react-redux-capstone/assets/59707859/a95591cb-a019-4732-87dc-1084701847ff)

![image (3)](https://github.com/martinkarugaba/react-redux-capstone/assets/59707859/7df175a4-52ec-4c2c-b108-e54d751ca0d6)